xl2times/transforms.py

```diff
@@ -2227,78 +2321,19 @@ def apply_transform_tables(
     Include data from transformation tables.
     """
 
-    topology = generate_topology_dictionary(tables, model)
-
-    def query(
```

Comment:
query() and eval_and_update() are above - I just moved them out of this function so I can call from test_transforms() for the comparison